subprojects/soak/soak.gradle

```diff
@@ -17,5 +17,5 @@ task('soakTest', type: org.gradle.testing.IntegrationTest) {
     options {
         includeCategories 'org.gradle.soak.categories.SoakTest'
     }
-    dependsOn({ project(':toolingApi').publishLocalArchives }, ':distributions:binZip')
+    dependsOn({ project(':toolingApi').publishLocalArchives }, ':distributions:testBinZip')
```

Comment:
Isn't this the same as setting `requiresBinZip = true` and `requiresLibsRepo = true`?